spectral_cube/cube_utils.py

```diff
@@ -191,8 +191,13 @@ def __iter__(self):
         raise Exception("You need to specify a slice (e.g. ``[:]`` or "
                         "``[0,:,:]`` in order to access this property.")
 
-def is_huge(cube, threshold=1e8):
-    if cube.size < threshold:  # smallish
+
+# TODO: make this into a proper configuration item
+# TODO: make threshold depend on memory?
```

Comment:
I think it's reasonable to *not* make the threshold depend on memory, but make this a configuration item.  Even with lots of memory, operations on huge cubes can take forever, and I think it's better to be up front about that.